### ISSUE TITLE
Remove delete

### ISSUE THREAD

**Issue by michaelBenin** [lukasolson/backbone-super#45]
https://github.com/lukasolson/backbone-super/blob/master/backbone-super/backbone-super.js#L57

Delete is a major perf hit:

http://kendsnyder.com/chrome-v8-creators-with-and-delete-are-dog-slow/

**Comment by philfreo** [comment]
What do you suggest instead?

**Comment by michaelBenin** [comment]
So before I found this project I was using:

https://coderwall.com/p/xj81ua/a-better-way-of-extending-backbone-models-and-views

And fixed it in my hobby project: 

https://github.com/michaelBenin/node-startup/blob/master/shared/utils/util.js#L5

If this was added would be a major version bump.

**Comment by apsavin** [comment]
@michaelBenin Can you provide a benchmark that it's really affect us in this particular case?

**Comment by michaelBenin** [comment]
@apsavin yes a perf comparison on the pr that removes the delete would be useful.

**Comment by michaelBenin** [comment]
@lukasolson when you get a chance maybe you could weigh in here.
